### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kimosop-Wakanda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Sudofy
 <img src="https://user-images.githubusercontent.com/54077752/110249544-990cd680-7f87-11eb-8300-dc5da1b9fb58.png"/>
 
-A project for Google Solution Challenge - 2021 (`Health monitoring system`)
+A project the for Google Solution Challenge - 2021 (`Health monitoring system`)
```
Comment (Ericgacoki) [inline README.md]: Hey @Kimosop-Wakanda, the grammar in this sentence doesn't flow so well, please correct that.

**Review by Ericgacoki — CHANGES_REQUESTED** [review]
Please make the changes ASAP.

**Review by Ericgacoki — APPROVED** [review]
checked

**Comment by Ericgacoki** [comment]
Nice work @Kimosop-Wakanda, next time please `ping` the reviewer when you're done with the changes.

**Comment by Kimosop-Wakanda** [comment]
Okay and
Good Luck with the project

**Pull request merged**